operations/services.md

```diff
@@ -1579,6 +1579,314 @@ Returns all companionships based on customers, reservations or reservation group
 | `ReservationId` | string | optional | Unique identifier of [Reservation](reservations.md#reservation). |
 | `ReservationGroupId` | string | required | Unique identifier of [Reservation group](reservations.md#reservation-group). |
 
+## Get all resource access tokens
+
+Returns all resource access tokens based on resource access tokens, reservations or interval. One of them must be specified in the request.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/resourceAccessTokens/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "ServiceIds": [
+        "bd26d8db-86da-4f96-9efc-e5a4654a4a94"
+    ],
+    "ResourceAccessTokenIds": [
+        "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D"
+    ],
+    "ReservationIds ": [
+        "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D"
+    ],
+    "CollidingUtc": {
+        "StartUtc": "2020-02-15T00:00:00Z",
+        "EndUtc": "2020-02-20T00:00:00Z"
+    },
+    "ActivityStates": [
+        "Active"
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of [Service](#service)s where the resource access tokens belong to. |
+| `ResourceAccessTokenIds` | array of string | optional, max 1000 items | Unique identifiers of [Resource access token](#resource-access-token)s. Required if no other filter is provided. |
+| `ReservationIds` | array of string | optional, max 1000 items | Unique identifiers of [Reservation](reservations.md#reservation)s. Required if no other filter is provided. |
```

Comment:
Now I've realized that everywhere else we have `ServiceOrder` and here we have `Reservations`, that's not great. Change it in code as well? cc @patrikmiksovsky95